docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# "docker-compose up" to start the services
```

Comment:
`docker-compose up app` if you only want to see the app logging, mongo service will still start due to depends_on